lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -36,14 +36,19 @@ class ConversionException extends \Doctrine\DBAL\DBALException
      *
      * @param string $value
      * @param string $toType
+     * @param \Exception|null $previous
      *
      * @return \Doctrine\DBAL\Types\ConversionException
      */
-    static public function conversionFailed($value, $toType)
+    static public function conversionFailed($value, $toType, \Exception $previous = null)
```

Comment:
I found a need for it when creating a custom doctrine type for PSR7 uris : I would like to get the message from the library that parses the string into an UriInterface object if it fails.